src/hostapi/dsound/pa_win_ds.c

```diff
@@ -1733,7 +1733,12 @@ static void CalculateBufferSettings( unsigned long *hostBufferSizeFrames,
     unsigned long maximumPollingPeriodFrames = (unsigned long)(sampleRate * PA_DS_MAXIMUM_POLLING_PERIOD_SECONDS);
     unsigned long pollingJitterFrames = (unsigned long)(sampleRate * PA_DS_POLLING_JITTER_SECONDS);
 
+<<<<<<< HEAD
```

Comment:
This merge conflict needs to be resolved before submitting.

Reply:
Sorry Phil, I was in a rush and did not follow my standard procedure:

1. Save the file before commit
2. Check the diff before commit
3. Check the diff before submitting the PR
4. Check the diff after submitting the PR
5. Check the CI results after submitting the PR

It should be OK now. The force pushes (1) fix the merge conflicts, (2) fix a missing `{`, (3) improve the short commit message.